Simplified/WelcomeScreen/NYPLWelcomeScreenViewController.swift

```diff
@@ -184,8 +184,16 @@ import PureLayout
       return
     }
     
-    let pickLibrary = {
-      let listVC = NYPLWelcomeScreenAccountList { account in
+    let presentLibraryFinder = {
+      var userAccounts = [Account]()
+      if let simplyEAccount = AccountsManager.shared.account(AccountsManager.NYPLAccountUUIDs[2]) {
```

Comment:
Are we absolutely sure that we're always gonna have a value for `AccountsManager.NYPLAccountUUIDs[2]`?

Reply:
Yeah, `AccountsManager.NYPLAccountUUIDs` is a static property we defined in code, so it should always have the same value unless any of us changes it.

Reply:
Definitely agree, this seems dangerous... I think the `NYPLAccountUUIDs` was a quick hack in the first place (I believe originally it was a private array), and what we could do maybe is refactor it so that we are asking things more semantically perhaps? Like "give me the SimplyE account number" instead of give me the third account.

To Ernest point though it's a bit out of scope of this PR, so @ErnestFan  feel free to file a ticket.

Reply:
I have added a property `SimplyEAccountUUID` in `AccountsManager`, can you take a second look?

Reply:
Looks better. Thanks